openfl/interface/collaborator_manager.py

```diff
@@ -19,6 +19,8 @@
 from openfl.component.collaborator_manager.collaborator_manager import CollaboratorManager
 from openfl.interface.cli_helper import SITEPACKS
 from openfl.interface.cli_helper import WORKSPACE
+from openfl.utilities.ca import download_step_bin  
```

Comment:
Seems like  `openfl.utilities.ca` is not included in the comment

Reply:
solve 05892bd4f2a756a40fd3af9b3ca8c71520d3e336